### DISCUSSION TITLE
lucky dev support hook when build success/failed.

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1798] [Q&A] [answered]
Probably a example is the best explain. check [this](https://forum.crystal-lang.org/t/make-sentry-support-play-wav-file-when-build-success-failed-can-anyone-help-on-try-it-on-macos/4840/3?u=zw963)

What i expected is a hook to run a bash script when build success or failed, or at least, can we override the default command which lucky dev will run as following sentry -b does?

```
 ╰─ $  sentry --help
Usage: ./sentry [options]
    -b COMMAND, --build=COMMAND      Overrides the default build command (will override --src flag)
```

Thanks

**Comment by jwoertink** [comment, marked as answer]
All `lucky dev` does is run 
https://github.com/luckyframework/lucky_cli/blob/7c3c69c775ba9aad7ac0206c052db66a906941c1/src/lucky.cr#L28-L29
https://github.com/luckyframework/lucky_cli/blob/7c3c69c775ba9aad7ac0206c052db66a906941c1/src/dev.cr#L5 

From there, it'll run whatever is in your `Procfile.dev` which is usually `lucky watch`. If you want a different command to run, you can update the Procfile.dev with anything else.
